assembler_tools/hec-assembler-tools/assembler/stages/prep/preprocessor.py

```diff
@@ -150,7 +150,9 @@ def reduce_var_deps_by_var(mem_model: MemoryModel, insts_list: list, var_name: s
         last_pos += 1
 
 
-def assign_register_banks_to_vars(mem_model: MemoryModel, insts_list: list, use_bank0: bool) -> str:
+def assign_register_banks_to_vars(
+    mem_model: MemoryModel, insts_list: list, use_bank0: bool, verbose=False, strategy="largest_first", interchange=False
```

Comment:
verbose parameter is not needed anymore as GlobalConfig verbosity is used instead.